internal/gatewayapi/helpers.go

```diff
@@ -369,6 +365,10 @@ func irTCPRouteName(route RouteContext) string {
 	return fmt.Sprintf("%s/%s/%s", strings.ToLower(string(GetRouteType(route))), route.GetNamespace(), route.GetName())
 }
 
+func irUDPRouteName(route RouteContext) string {
+	return fmt.Sprintf("%s/%s/%s", strings.ToLower(string(GetRouteType(route))), route.GetNamespace(), route.GetName())
```

Comment:
can we just plug in `return irTCPRouteName(route)` here ? to reduce code duplication ?

Reply:
sure.